docker/package/packages.py

```diff
@@ -136,18 +143,24 @@
                                                   config_file="tezos-baker.conf")],
                                      proto,
                                      optional_opam_deps=["tls", "ledgerwallet-tezos"],
-                                     requires_sapling_params=True))
+                                     requires_sapling_params=True,
+                                     postinst_steps=postinst_steps_common + '''
+mkdir -p /var/lib/tezos/baker'''))
     packages.append(OpamBasedPackage(f"tezos-accuser-{proto}", "Daemon for accusing",
                                      [SystemdUnit(service_file=service_file_accuser,
                                                   startup_script="tezos-accuser-start",
                                                   config_file="tezos-accuser.conf")],
                                      proto,
-                                     optional_opam_deps=["tls", "ledgerwallet-tezos"]))
+                                     optional_opam_deps=["tls", "ledgerwallet-tezos"],
+                                     postinst_steps=postinst_steps_common + '''
+mkdir -p /var/lib/tezos/accuser'''))
     packages.append(OpamBasedPackage(f"tezos-endorser-{proto}", "Daemon for endorsing",
                                      [SystemdUnit(service_file=service_file_endorser,
                                                   startup_script="tezos-endorser-start",
                                                   config_file="tezos-endorser.conf")],
                                      proto,
-                                     optional_opam_deps=["tls", "ledgerwallet-tezos"]))
+                                     optional_opam_deps=["tls", "ledgerwallet-tezos"],
+                                     postinst_steps=postinst_steps_common + '''
+mkdir -p /var/lib/tezos/endorser'''))
```

Comment:
So looking back at the part of the article that originated this issue I think there's still two things that are required from the user (and probably shouldn't):

1, the endorser and baker have two different `DATA_DIR`, which could be fixed by changing here and in `docker/package/defaults/tezos-endorser.conf`. Although there may be some downside for doing that?
2. If I'm not mistaken it's still necessary to set the `NODE_DATA_DIR` for the baker manually since it's not specified in `docker/package/defaults/tezos-baker.conf`, but since we are now using and creating all the default dirs, I suppose that should be changed?

Reply:
1) Well, pointing both baker and endorser to the same `/var/lib/tezos/baker` could be confusing, because AFAIU, endorser can run without a baker and vice versa.
2) Yes, because usually there are at least two networks that one might use: mainnet and testnet that correspond to the current mainnet. So I propose to require the user to set the `NODE_DATA_DIR` manually so that he knows which network he's using

Reply:
Regarding 1.: maybe it's worth pointing them to a single directory called `/var/lit/tezos/client` and mention that this directory stores all keys related information

Reply:
Ok, regarding 2, that's a good point, I agree :ok_hand: 

Actually the same kind of reasoning can be applied to the first point, as indeed the `backer` and `endorser` don't have to run on the same keys and there are situations where that could be used.

So I think even there the user should be left to change that in the tutorial himself.

No change required, all things considered :+1: